vulnerabilities/importers/apache_kafka.py

```diff
@@ -7,120 +7,177 @@
 # See https://aboutcode.org for more information about nexB OSS projects.
 #
 
-import asyncio
 
 import requests
 from bs4 import BeautifulSoup
 from packageurl import PackageURL
-from univers.version_range import VersionRange
+from univers.version_constraint import VersionConstraint
+from univers.version_range import MavenVersionRange
 from univers.versions import MavenVersion
 
 from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import AffectedPackage
 from vulnerabilities.importer import Importer
 from vulnerabilities.importer import Reference
-from vulnerabilities.package_managers import GitHubTagsAPI
-from vulnerabilities.utils import nearest_patched_package
 
-GH_PAGE_URL = "https://raw.githubusercontent.com/apache/kafka-site/asf-site/cve-list.html"
-ASF_PAGE_URL = "https://kafka.apache.org/cve-list"
+# The entries below with `"action": "omit"` have no useful/reportable fixed or affected version data.
+# See https://kafka.apache.org/cve-list
+affected_version_range_mapping = {
+    "CVE-2022-34917": {
+        "action": "include",
+        "2.8.0 - 2.8.1, 3.0.0 - 3.0.1, 3.1.0 - 3.1.1, 3.2.0 - 3.2.1": "affected",
+        "2.8.2, 3.0.2, 3.1.2, 3.2.3": "fixed",
+        "affected_version_range": "vers:maven/>=2.8.0|<=2.8.1|!=2.8.2|>=3.0.0|<=3.0.1|!=3.0.2|>=3.1.0|<=3.1.1|!=3.1.2|>=3.2.0|<=3.2.1|!=3.2.3",
```

Comment:
Should this be `vers:apache` or `vers:maven` ?

Reply:
Thanks @TG1999 -- I've changed `vers:maven` to `vers:apache`.